### PR TITLE
Add imagestream generator and YAML config for this generator

### PR DESCRIPTION
**Pull request by phracek**

### CODE REVIEW

**Comment by phracek** [comment]
[test-openshift]

**Comment by phracek** [comment]
This pull request blocks #370

**Comment by phracek** [comment]
Let's have only one commit. It does not matter for review to have splitter into more commits. Let's finish the imagestream generator part across all repositories.

File: imagestreams/mysql-rhel.json [hunk 1]
```diff
@@ -63,36 +46,36 @@
...
           "kind": "DockerImage",
-          "name": "registry.redhat.io/rhscl/mysql-80-rhel7:latest"
+          "name": "registry.redhat.io/rhel9/mysql-80:latest"
         },
         "referencePolicy": {
           "type": "Local"
         }
       },
       {
-        "name": "8.0",
```
Comment (zmiklank) [inline imagestreams/mysql-rhel.json:84]: This is a nonstandard tag, it should be added to the config like this: https://github.com/sclorg/httpd-container/blob/19edd655824b9d03d4a6e0d509fce02fdaf0c457/imagestreams/imagestreams.yaml#L38-L41

Comment (phracek) [reply imagestreams/mysql-rhel.json:84]: Fixed by https://github.com/sclorg/mysql-container/pull/371/commits/17717fbe5b2ff1838a81adf88859e1542dc47b78

**Comment by phracek** [comment]
[test-openshift]

**Review by zmiklank — APPROVED** [review]

**Pull request merged**